apps/spruce/src/components/PatchesPage/index.tsx

```diff
@@ -97,33 +86,25 @@ export const PatchesPage: React.FC<Props> = ({
           onChange={includeHiddenCheckboxOnChange}
         />
       </FiltersWrapperSpaceBetween>
-      <PaginationRow>
-        <Pagination
-          currentPage={page}
-          pageSize={limit}
-          totalResults={patches?.filteredPatchCount ?? 0}
-        />
-        <PageSizeSelector
-          data-cy="my-patches-page-size-selector"
-          onChange={handlePageSizeChange}
-          value={limit}
-        />
-      </PaginationRow>
+      <PaginationButtons
+        filteredPatchCount={patches?.filteredPatchCount}
+        pageType={pageType}
+      />
       <ListArea
         loading={loading}
         pageType={pageType}
         patches={patches?.patches || []}
       />
+      {!loading && (
+        <PaginationButtons
+          filteredPatchCount={patches?.filteredPatchCount}
+          pageType={pageType}
+        />
```

Comment:
On the version page we conditionally show the pagination buttons if the length of the page is > 10 should we do that here?